code/game/world.dm

```diff
@@ -277,27 +277,29 @@ GLOBAL_VAR(restart_counter)
 	s += "Forums"
 	s += "</a>" //yog end
 	s += ")"
-	
+
 	var/players = GLOB.clients.len
-	
+
 	var/popcaptext = ""
 	var/popcap = max(CONFIG_GET(number/extreme_popcap), CONFIG_GET(number/hard_popcap), CONFIG_GET(number/soft_popcap))
 	if (popcap)
 		popcaptext = "/[popcap]"
-	
+
 	if (players > 1)
 		features += "[players][popcaptext] players"
 	else if (players > 0)
 		features += "[players][popcaptext] player"
-	
+
 	game_state = (CONFIG_GET(number/extreme_popcap) && players >= CONFIG_GET(number/extreme_popcap)) //tells the hub if we are full
-	
+
 	if (!host && hostedby)
 		features += "hosted by <b>[hostedby]</b>"
 
 	if (features)
 		s += ": [jointext(features, ", ")]"
 
+	s += "<br>[pick(world.file2list("yogstation/strings/taglines.txt"))]<br>" //yogs - server tagline, ported from citadel

```

Comment:
do this in the mirrored version of the function

Reply:
I also removed the gamemode, because it's literally just gonna be secret about 97% of the time, the other 3% being battle royale, at which point you can't latejoin anyways.